### DISCUSSION TITLE
JavaMelody with SPA (Angular 11)

### DISCUSSION THREAD

**Discussion by mguinchard** [javamelody/javamelody#1013] [General]
Hello,

I want to use JavaMelody on an SPA (frontend Angular 11 and backend spring-boot).

Can JavaMelody be used in this architecture ?
(I have some routing problems currently !

 Thank you for your answers.

Maxime G.

**Comment by evernat** [comment]
Yes. It happens that I currently work every day with an application using a frontend Angular 11 and a backend spring-boot (embedded Tomcat). It includes just a javamelody-spring-boot-starter dependency 😄 (no surprise) with only a sql-transform-pattern parameter. There is no routing problem and no routing or web configuration for that.
You may describe the issue and how to reproduce it and what you expect, and give if possible a reduced source of a project to reproduce it.
